crates/miden-tx/src/testing/mock_chain/mod.rs

```diff
@@ -897,6 +912,9 @@ impl MockChain {
                 self.pending_objects.updated_accounts.clone(),
                 self.pending_objects.output_note_batches.clone(),
                 self.pending_objects.created_nullifiers.clone(),
+                // TODO: For now we can't easily compute the verified transactions of this block.
+                // Let's do this as part of miden-base/#1244.
+                vec![],
```

Comment:
Is the reference #1244 correct here? Seems like it is already merged and has to do with open telemetry.

Reply:
Impressive catch :smile: Yes, I typo'd - it should have been #1224.